config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -499,12 +505,163 @@ periodics:
         requests:
           cpu: 2
           memory: 6Gi
-  annotations:
-    testgrid-dashboards: sig-release-master-blocking, google-gce, google-gci
-    testgrid-tab-name: gce-cos-master-default
-    testgrid-num-failures-to-alert: '6'
-    testgrid-alert-email: release-team@kubernetes.io
-    description: Uses kubetest to run e2e tests (-Slow|Serial|Disruptive|Flaky|Feature) against a cluster created with cluster/kube-up.sh
+      env:
+      - name: ENABLE_APISERVER_ADVANCED_AUDIT
+        value: "true"
+      - name: ADVANCED_AUDIT_POLICY
```

Comment:
I don't think we should change the default audit policy for the default e2e job

Reply:
Currently we need the results of a full test run (for all tests, not just the conformance runs, so we can see tests that we could promote, or emulate to hit with conformance tests.

Is there another job we could modify that has similar perodicity that's not the default job?

Reply:
I'm fine leaving it alone for now. It means we might be lacking non-conformance test coverage, but it's not as important as conformance coverage itself.